src/main/java/edu/cornell/kfs/concur/batch/ConcurRequestExtractPdpConstants.java

```diff
@@ -0,0 +1,23 @@
+package edu.cornell.kfs.concur.batch;
+
+public class ConcurRequestExtractPdpConstants {
```

Comment:
With bulk of the code in this file being placed in other files, I will also be moving the two remaining constants to the ConcurConstants file and totally deleting this constants class.